R/plot-build.r

```diff
@@ -432,3 +359,88 @@ by_layer <- function(f, layers, data, step = NULL) {
   )
   out
 }
+
+# Add the tag element to the gtable
+table_add_tag <- function(table, label, theme) {
+  # Initialise the tag margins
+  table <- gtable_add_padding(table, unit(0, "pt"))
```

Comment:
Patchwork for some reason assumes this padding is present regardless of whether it was needed, so I left this in.

Reply:
yes, thanks... We should generally try to be predictable wrt the gtable structure that ggplot2 produces as it makes aligning multiple plots easier for patchwork

Reply:
That makes good sense to me :)